astropy/io/votable/tree.py

```diff
@@ -2123,7 +2123,7 @@ def get_ref(self):
         references.
         """
         for field in self._table._votable.iter_fields_and_params():
-            if isinstance(field, Field) and field.ID == self.ref:
+            if isinstance(field, Field) and self.ref == field.ID:
```

Comment:
This change is in the wrong direction, and really quite illogical: mind is on field, so makes sense to have it first.